### PR TITLE
Merge bsc branch

### PR DESCRIPTION
**Pull request by karim-en**
This pull request merges the [bsc](https://github.com/aurora-is-near/rainbow-bridge/tree/bsc) branch with `master` and  improves deploy-related commands in the Makefile

### CODE REVIEW

**Comment by idirall22** [comment]
hi

**Review by idirall22 — CHANGES_REQUESTED** [review]

**Comment by karim-en** [comment]
@idirall22 thanks for the review, I've handled your request

**Review by sept-en — APPROVED** [review]

File: near2eth/near2eth-block-relay/index.js [hunk 1]
```diff
@@ -70,7 +71,7 @@ class Near2EthRelay {
         // The finalized block is not immediately available so we wait for it to become available.
         let lightClientBlock = null
         let currentValidators = null
-        while (!lightClientBlock || Object.keys(lightClientBlock).length === 0) {
+        while (_.isEmpty(lightClientBlock)) {
```
Comment (sept-en) [inline near2eth/near2eth-block-relay/index.js:74]: I would prefer not to have this `_.METHOD_NAME` pattern in our code. Please change

Comment (karim-en) [reply near2eth/near2eth-block-relay/index.js]: fixed at 77ef641809463892d74d6e494ffc0bf607298ed0

**Review by sept-en — APPROVED** [review]

**Pull request merged**